docs/source/noise/index.rst

```diff
@@ -178,9 +178,10 @@ Noise types for each column
     - Leave a field blank, choose the wrong option
     -
   * - SSN
-    - W-2 and 1099, SSA
-    - Borrow a social security number, leave a field blank, write the wrong digits, make typos, make OCR errors
-    - Note that 'borrow a social security number' only applies to the W-2 and 1099 dataset
+    - W-2 and 1099, 1040, SSA
+    - "Borrowed" SSN, leave a field blank, write the wrong digits, make typos, make OCR errors
+    - Note that "borrowed" SSN only applies to the W-2 and 1099 dataset.
```

Comment:
This is missing the changes made in https://github.com/ihmeuw/pseudopeople/pull/237.